File: Python/supportLib.py
import array

def DoubleArrayToByteArray(data):
	ints = [int(x) for x in data]
	bytes = array.array('B', ints)
	return bytes;

def WriteToFile(data, filename):
	f = open(filename, 'w+b')
	bytes = DoubleArrayToByteArray(data)
	f.write(bytes)
	f.close()

	return

File: TypeScript/supportLib.ts
import * as fs from 'fs';

export function ToByteArray(data : number []) : Uint8Array{
	let bytes : Uint8Array = new Uint8Array(data.length);

	for(var i = 0; i < data.length; i++){
		bytes[i] = data[i];
	}

	return bytes;
}

export function WriteToFile(data : number [], filename : string){
	var bytes = ToByteArray(data);
	var wstream = fs.createWriteStream(filename);
	wstream.write(bytes);
	wstream.end();
}
